### PR TITLE
CB-12511 added DeprecatedValidator to blueprint url based submission

### PR DESCRIPTION
**Pull request by lnardai**
New error message when submitting URL for BPs
`Submitting Cluster Template by URL is not allowed anymore. Please use text based submission.
You have submitted a field that is not accepted anymore. This is usually due to deprecation or security reasons.`

Please do not MERGE until @aszegedi is finished with the Blueprint changes for the GUI tests.

### CODE REVIEW

**Comment by lacikaaa** [comment]
converted to draft to prevent accidental merge

File: core-api/src/main/java/com/sequenceiq/cloudbreak/validation/DeprecatedValidator.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.cloudbreak.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class DeprecatedValidator implements ConstraintValidator<ValidDeprecated, String> {
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (value == null || value.isEmpty()) {
```
Comment (lacikaaa) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/DeprecatedValidator.java:10]: you could use `StringUtils#isBlank`

File: core-api/src/main/java/com/sequenceiq/cloudbreak/validation/DeprecatedValidator.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            return true;
+        }
```
Comment (lacikaaa) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/DeprecatedValidator.java:12]: I prefer having an `else` branch. Makes the code indentation more obvious about the exit point of the method

**Comment by aszegedi** [comment]
I've done on [CB-12496 Remove URL based resource creation tests #2765](https://github.com/hortonworks/hortonworks-cloud/pull/2765). Tests are still in progress...

**Comment by lnardai** [comment]
CB-12496 Remove URL based resource creation tests Has been merged.

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**